client/comms/wsconn.go

```diff
@@ -564,7 +564,7 @@ func (conn *wsConn) Connect(ctx context.Context) (*sync.WaitGroup, error) {
 		close(conn.readCh) // signal to MessageSource receivers that the wsConn is dead
 	}()
 
-	return &conn.wg, err
+	return &conn.wg, nil
```

Comment:
It should be up to the caller to decide how to handle `Connect` errors. Maybe a const error?

Reply:
Oh. I see now. My bad.

Reply:
Ok? From the comments I think the intent was to keep going on error, we could make an exception inside the callers (dex.runner) but I don't think necessary, as all we would do is log the error again.

Reply:
I think `nil` is right. Ancient bug.